Add Mappers' Report: May news post (#5720)
ppy/osu-wiki: news/2021-06-18-mappers-report-may.md

---
layout: post
title: "The Mappers' Report: May"
date: 2021-06-18 08:00:00 +0000
---

Take everything in with the May issue of the Mappers' Report and begin your summer up-to-date with the mapping and modding community!

![](/wiki/shared/news/banners/the-mappers-report.jpg)

This month's issue of the Mappers' Report contains the following topics:

- **[Ranking Criteria Changes](#ranking-criteria-changes)**
- **[Within the Beatmap Nominators and NAT](#within-the-beatmap-nominators-and-nat)**
- **[Mappers' Showcase](#mappers-showcase)**
- **[Mapping Contests](#mapping-contests)**

> - **[Around the Community](#around-community)**

Reviewer: ```suggestion
- **[Around the Community](#around-the-community)**
```

Well, how does it sound? Have we got you interested in getting your hands dirty with the fruit juice? Then be quick, the sign-ups closing soon on the **20th June**!

### osu!catch Mapping World Cup 2021

Another contest that is going to fill your fruit bowl has been brought up to light, and this time a country-based one! Brought to you by the osu!catch mapping grand-master [Ascendance](https://osu.ppy.sh/users/2931883) for the third time, the [osu!catch Mapping World Cup 2021](https://osu.ppy.sh/community/forums/topics/1350409) will have the bestest mappers from all over the globe gather around and have an intense competition that will determine the next osu!catch Mapping Champions!

Needless to say that such a contest isn't going to be left without prizes, but all the details for that matter are currently work-in-progress. To keep track of the updates and learn more about the contest, we highly suggest you to check the aforementioned forum thread, and dive right into the [o!CMWC Discord server](https://discord.gg/eTUFAjf "Discord") prepared with love for this very contest!

> Alright catch folks! Hopefully, we've made your day with these two majestic contests, but be warned that they may over the dose of your daily fruit intake. Proceed with caution!

Author: I am pretty sure the pun with "overdose", which is a difficulty name for osu!catch beatmapsets is intended here :p

## Around the Community

> The community has worked hard this month to present you with a variety of activities that happened in the past month. Got you interested? Awesome! Then continue reading to find out more.

Reviewer 2: variety

```suggestion
The community has worked hard this month to present you with a variety of activities that happened in the past month. Did that strike your fancy? Awesome! Then continue reading to find out more.
```

### Guide to making your first osu!mania 10K beatmap

Not so long ago, osu!mania got a new key-mode. After a long wait, 10K beatmaps can finally be ranked. And that's exactly what they need now. But how to start and how to map 10K? The same question came up on the 10K+ Rhythm Game Community server and so they gathered and created their own guide!

The 10K community representative [\[Sick\]Catcat](https://osu.ppy.sh/users/11544934) has posted their [10K osu!mania Mapping Guide](https://osu.ppy.sh/community/forums/topics/1336072) in the osu!mania sub forum with everything you need to know about mapping 10K osu!mania. From how to start, various pattern structures to useful tips and tricks from Easy to Hard difficulties.

> Got you interested? That's great! Then get your favourite song and start mapping. 10K osu!mania is in urgent need of more ranked beatmaps. And should you need more help, the [10K+ Rhythm Game Community](https://discord.gg/PwzcUzk) would be more than happy to help you out further with your map!

Reviewer 2: variety

```suggestion
If it piqued your interest, that's great! Then get your favourite song and start mapping. 10K osu!mania is in urgent need of more ranked beatmaps. And should you need more help, the [10K+ Rhythm Game Community](https://discord.gg/PwzcUzk) would be more than happy to help you out further with your map!
```

### osu! Featured Artist Compilation Blind Beatmap Collab

Now, let's take a moment for ourselves and admire the wonderful collaborative effort the community has put forward, because [**this is where we are, and what unites us**](https://osu.ppy.sh/beatmapsets/1483077#osu/3041896).

> Hosted by the talented mapper and modder [Nuvolina](https://osu.ppy.sh/users/10974170), the blind collab project features an hour-long medley composed of minute sections from various Featured Artist tracks delicately compiled by [ModlesS\_Flash](https://osu.ppy.sh/users/5113095).

Author: That really should be visible enough as it is. Links are pretty visible

And of course, such a massive project wouldn't have been left without a showcase stream! The vod is available below for those who have missed it or wish to fall under its spell once again:

<iframe src="https://player.twitch.tv/?video=1051031340&parent=osu.ppy.sh" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>

<a id="conclusion"></a>

---

> And that’s it for this month. Good we didn’t promise anything last time because once again we are late but the IRL stress will finally be gone within the next weeks which means the next issues should be hopefully on time!

Reviewer 2: ```suggestion
And that’s it for this month. Good we didn’t promise anything last time because we are once again late, but the IRL stress will finally be gone within the next weeks, which means the next issues should hopefully be on time!
```
